cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1313,6 +1313,17 @@
                       be appropriate if following the pattern
                       ``hosts = main, backup, failsafe``.
                 ''')
+            with Conf('directives', desc='''
+                Job runner (batch scheduler) directives.
+
+                Default directives for :cylc:conf:
+                `flow.cylc[runtime][<namespace>][directives]`
```

Comment:
Looks like the precedent for this kind of thing is
```suggestion
                :Defaults for: :cylc:conf:`flow.cylc[runtime][<namespace>][directives]`
```
(avoiding duplicating the description)

Reply:
Fair enough, in that case
```suggestion
                :Defaults for: :cylc:conf:`flow.cylc[runtime][<namespace>][directives]`
                
                Job runner (batch scheduler) directives.
```
`:Defaults for:` is some kind of sphinx directive by the look of it, possibly from cylc-sphinx-extensions